### ISSUE TITLE
Urban Sports Club App QR-Code Check In

### ISSUE THREAD

**Issue by leiaorganaskywalker** [GrapheneOS/os-issue-tracker#4474]
I ran into an issue with the Urban Sports Club App. Currently I am using Version 9676 from Aurora Store. The App is set up in a second profile with Google Play Services installed. The app mostly runs fine, except during check-in for a lesson.

For the purpose of check-in the app features it's own built-in QR Code scanner. It opens up, but does not scan any code. The app does not crash, it simply does not react at all. I tried possible workaround solutions to enhance compatibility according to GoS Forum, but nothing worked.

[Urban Sports Club log badbf6665599.txt](https://github.com/user-attachments/files/18075180/Urban.Sports.Club.log.badbf6665599.txt)

**Comment by MDC91** [comment]
Have you found a solution for the issue? I have the exactly same problem.

**Comment by leiaorganaskywalker** [comment]
No unfortunately I didn't. Currently I am using an iPhone running iOS for the check in process.
I also was not aware of blocking any connections for Play Store or Play Services, since I didn't use a VPN or firewall. Maybe @thestinger has any ideas how we can solve the issue?

**Comment by muhomorr** [comment]
@leiaorganaskywalker Does Play Store have the Network permission? Play Store needs it to download the QR code scan module.

**Comment by leiaorganaskywalker** [comment]
No. I just granted this permission to Play Store. Have to check next time, when I go and work out 🏋

**Comment by leiaorganaskywalker** [comment]
> Play Store needs it to download the QR code scan module.

Yep, this is the solution. It's working now!
Sorry for inconvenience, I just was not aware that I didn't grant the network permission to Play Store.
Thanks for your help!
